examples/mnist_test_oss.py

```diff
@@ -0,0 +1,169 @@
+## adapted from https://github.com/pytorch/examples/blob/master/mnist/main.py
+from __future__ import print_function
+import argparse
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.optim.lr_scheduler import StepLR
+
+import time
+from fairscale.optim.oss import OSS
+import torch.multiprocessing as mp
+from fairscale.nn.data_parallel import ShardedDataParallel
+
+# how is world_size defined?
```

Comment:
depends on the number of (number of machines in the gang) * (number of gpus per machine). Can be just 2 if you're on a 2-GPU machine

Reply:
(2 is essentially the minimum viable example, hence it was being used in the other tutorial)